src/seedu/addressbook/AddressBook.java

```diff
@@ -140,10 +154,23 @@
     private static final int PERSON_DATA_INDEX_PHONE = 1;
     private static final int PERSON_DATA_INDEX_EMAIL = 2;
 
+    /* We use a String array to store details of an edit command.
+     * The constants given below are the indexes for the different data elements of an edit command
+     * used by the internal String[] storage format.
+     * For example, a person's index to be edited is stored as the 0th element in the array.
+     */
+    private static final int EDIT_INDEX_INDEX_NUMBER = 0;
+    private static final int EDIT_INDEX_ATTRIBUTE = 1;
+    private static final int EDIT_INDEX_NEW_VALUE = 2;
+    
+    private static final String[] EDIT_ATTRIBUTES = {"name", "phone", "email"};
```

Comment:
Good that you try to group them here as constants. Read on "enums" in the handouts. Perhaps you may find it more convenient for this :]